fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneEvents.java

```diff
@@ -268,4 +256,40 @@ public boolean isSize() {
             return isSize;
         }
     }
+
+    /**
+     * Size Events.
+     */
+    public enum SizeEvents implements StoreTimer.SizeEvent {
+
+        /** writeFileReference operation in the FDBDirectory.*/
+        LUCENE_WRITE_FILE_REFERENCE("lucene write file references"),
+        /** writeData operation in FDBDirectory. */
+        LUCENE_WRITE("lucene index writes", true),
+        /** Write Stored Fields operations on the FDBDirectory. */
+        LUCENE_WRITE_STORED_FIELDS("lucene write stored fields"),
```

Comment:
In addition to these Lucene-related events, there are a few core Record Layer count events that I think it would make sense to update. For instance, we could probably remove/deprecate practically all of the events where there is both a `BLAH_BLAH` event and a `BLAH_BLAH_BYTES` event in favor of a single event. We don't necessarily need to do that in this PR, though

Reply:
Yeah, I see that there are lot many of that sort. I am not looking to do it in this PR but there is definitely a possibility of follow-up work that cleans up "size" metrics.